modules/fuse-box-responsive-api/index.js

```diff
@@ -9,10 +9,11 @@
 
     /* @if universal */
     var isBrowser = typeof window !== "undefined";
+    /* @if !npm */
     if (!isBrowser) {
```

Comment:
An npm library can be used server-side (with node) or client-side (bundled with fuse-box, webpack, etc.)
The only common part is that the engine using the npm library follows the `commonjs` standards and provide a `require` function to the library.

`commonjs` allows the "plug-n-play" saying that each libraries `require` other libraries and `module.exports` or `exports` values.

So, a npm library doesn't have to provide a `require` as the `require` will be provided by the engine (node/bundler/...), and this is valid agnostically of server/client side